src/server/game/Warden/WardenWin.cpp

```diff
@@ -239,6 +252,61 @@ void WardenWin::HandleHashResult(ByteBuffer& buff)
     _initialized = true;
 }
 
+uint16 WardenWin::GetFreePayloadId()
+{
+    uint16 payloadId = _wardenPayloadOffset;
+        
```

Comment:
Workflow says there is a whitespace here.

Reply:
Yup no problem I will fix it.

Reply:
This is fixed.